pkg/server/ptypes/deployment.go

```diff
@@ -81,3 +81,11 @@ func ValidateGetDeploymentRequest(v *pb.GetDeploymentRequest) error {
 		}),
 	))
 }
+
+// TODO: Make this generic w/ build + release?
```

Comment:
I wouldn't! I think it's okay we're extra verbose with these API validation functions.

Reply:
Sounds good to me!